package.json

```diff
@@ -4,7 +4,7 @@
   "publisher": "veelenga",
   "icon": "assets/ameba.png",
   "description": "Ameba extension for Visual Studio Code",
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
Please remove this change, as it's not in the scope of this PR.

Reply:
Apologies, was left there for testing (installing a dev version requires a newer version)